udt.go

```diff
@@ -39,11 +39,17 @@ func makeUDT(value reflect.Value, mapper *reflectx.Mapper, unsafe bool) udt {
 
 func (u udt) MarshalUDT(name string, info gocql.TypeInfo) ([]byte, error) {
 	value, ok := u.field[name]
-	if !ok {
-		return nil, fmt.Errorf("missing name %q in %s", name, u.value.Type())
+
```

Comment:
Should we consider `unsafe` flag ?
```
func (u udt) MarshalUDT(name string, info gocql.TypeInfo) ([]byte, error) {
	value, ok := u.field[name]
	if ok {
		return gocql.Marshal(info, value.Interface())
	}
	if u.unsafe {
		return nil, nil
	}
	return nil, fmt.Errorf("missing name %q in %s", name, u.value.Type())
}
```

Reply:
@dkropachev AFAIK this would require a user to use `query.Iter().Unsafe()` instead of just `query.Exec()`, because in `gocql` implementation there is no `unsafe` equivalent. Wouldn't it be counterintuitive if in `gocql` we are able to do `query.Exec()` with udt with missing fields and it works, but in `gocqlx` we are required to use `Unsafe()`?

Reply:
@sylwiaszunejko , 100% agree on that, but since Unsafe is there and does control this particular behavior we have to be consistent in the way it works.